apistrap/decorators.py

```diff
@@ -106,12 +107,18 @@ def __init__(self, swagger: 'apistrap.flask.Swagger', response_class: Type[Model
     def __call__(self, wrapped_func: Callable):
         _ensure_specs_dict(wrapped_func)
 
-        wrapped_func.specs_dict["responses"][str(self._code)] = {
-            "schema": {
-                "$ref": self._swagger.add_definition(self._response_class.__name__, self._get_schema_object())
-            },
-            "description": self._description
-        }
+        if self._response_class == FileResponse:
+            wrapped_func.specs_dict["responses"][str(self._code)] = {
+                "schema": {},
```

Comment:
Schema should be `{"type": "file"}` here. Also, it would be nice if we could set the description somehow.

Also, if we know the mimetype, it would be helpful to set `specs_dict["produces"]` to this mimetype.

Reply:
I'll fix the shema.

The `description` and `produces` are kindy tricky as the are known no sooner than in the wrapped function which seems late. Any hint on setting it from within?

One option could be having multiple responses such as  `CsvResponse` and `PdfResponse` providing these metas by some static methods/properties... but I don't like it much.

Reply:
You could specify the mime type and description in the decorator and then add it to the response object automatically.

Reply:
ok, will do :) thanks

Reply:
ok, this could do the trick